core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1552,6 +1553,13 @@ maybe_set_call_forward({Endpoint, Call, CallFwd, CCVs}) ->
                        )
     }.
 
+-spec maybe_rtcp_mux(ccv_acc()) -> ccv_acc().
+maybe_rtcp_mux({Endpoint, Call, CallFwd, CCVs} = Acc) ->
+    case kz_json:is_true([<<"media">>, <<"rtcp_mux">>], Endpoint) of
```

Comment:
`kz_json:is_true/2` will never return `undefined`
you need to use `kz_json:get_value/2` and then use `kz_term:to_boolean(RTCP_MUX)`
you can also add a `kz_json:get_boolean_value/2/3` like  `kz_json:get_atom_value/2/3` and others and use here without the need to use `kz_term`. `kz_json:is_true` and `kz_json:is_false` always return `true/false`